src/schema-gen/mvn-xsd-proofcheck-xpl.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
Minor, but blocking: Can you please mark this executable by running this from WSL2 on your computer? This makes it much easier to use for others and ready to bake in automated pipelines.

```sh
chmod +x ./src/schema-gen/mvn-xsd-proofcheck-xpl.sh
```

Reply:
I was able to push it myself, so you can disregard this pending review item and I will approve.